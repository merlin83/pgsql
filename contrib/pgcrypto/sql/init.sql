--
-- init pgcrypto
--

\set ECHO none
SET autocommit TO 'on';
\i pgcrypto.sql
\set ECHO all

-- check for encoding fn's
SELECT encode('foo', 'hex');
SELECT decode('666f6f', 'hex');

