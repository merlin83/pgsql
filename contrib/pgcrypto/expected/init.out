--
-- init pgcrypto
--
\set ECHO none
-- check for encoding fn's
SELECT encode('foo', 'hex');
 encode 
--------
 666f6f
(1 row)

SELECT decode('666f6f', 'hex');
 decode 
--------
 foo
(1 row)

